app/src/main/java/be/scri/fragments/MainFragment.kt

```diff
@@ -39,9 +42,34 @@ class MainFragment : ScribeFragment("Main") {
             }
         (requireActivity() as MainActivity).setActionBarButtonVisibility(false)
         callback.isEnabled = true
+        isArabicLanguage()
         return binding.root
     }
 
+    private fun isArabicLanguage(): Boolean {
+        val cornerTriangleImageView: ImageView = binding.cornerTriangle
+        val cornerCogImageView: ImageView = binding.cornerCog
+        val paramsTriangleLayout = binding.cornerTriangle.layoutParams as FrameLayout.LayoutParams
+        val paramsCogLayout = binding.cornerCog.layoutParams as FrameLayout.LayoutParams
+        val currentLocale =
+            requireActivity()
+                .resources.configuration.locales
+                .get(0)
+        val languageCode =
+            currentLocale
+                .language
+        if (languageCode == "ar") {
+            paramsTriangleLayout.gravity = Gravity.TOP or Gravity.START
+            paramsCogLayout.gravity = Gravity.TOP or Gravity.START
+            cornerTriangleImageView.scaleX = -1f
+            cornerCogImageView.scaleX = -1f
+            Log.d("Debug", "Arabic")
+        } else {
+            Log.d("Debug", "Not Arabic")
+        }
+        return languageCode == "ar"
+    }
```

Comment:
we need to make a single responsibility to this method, i.e. making it do single task. 

from it's name, 'isArabicLanguage' , which mean it check the language so it should not have logic related to change the UI configuration such as scale/gravity. 

so I think we would split it into two method, one to check language and other to change the view data. 

checking language would be in separate util function and would be unit tested.

Reply:
`isArabicLanguage` should also be `isRTLLanguage` so that we can have it be general from the start :)